src/main/java/com/ncc/aif/OntToJavaGen/OntologyGeneration.java

```diff
@@ -0,0 +1,185 @@
+package com.ncc.aif.OntToJavaGen;
+
+import org.apache.jena.ontology.OntModel;
+import org.apache.jena.ontology.OntClass;
+import org.apache.jena.ontology.OntModelSpec;
+import org.apache.jena.ontology.OntDocumentManager;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.util.FileUtils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Iterator;
+
+/**
+ * Generates java resources  from an ontology
+ */
+public class OntologyGeneration {
```

Comment:
Skip the @author tag since it doesn't look like we use it much on this project.  But the class comment should provide more information/instructions or refer to the README.

Reply:
Reference to README was added.